ironbeers-client-side/src/components/RandomBeer.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import Header from "./Header";
+import axios from "axios";
+
+class RandomBeer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { 
+      randomBeer: {}
+    };
+  }
+
+  getRandomBeer = () => {
+    axios
+      .get(`https://ironbeer-api.herokuapp.com/beers/random`)
+      .then(responseFromApi => {
+        this.setState({
+          randomBeer: responseFromApi.data[0]
```

Comment:
Tu aurais pu faire `this.setState(responseFromApi.data[0])` pour ne pas avoir à répéter `randomBeer` dans `this.state.randomBeer.image_url|name|...`